### PR TITLE
renaming keys

### PR DESCRIPTION
**Pull request by seeden**
first working version renaming keys

### CODE REVIEW

File: packages/api-react/src/services/daemon.ts [hunk 1]
```diff
@@ -1,23 +1,100 @@
...
+      }
+    >({
+      query: ({ fingerprint, label }) => ({
+        command: 'setLabel',
+        service: Daemon,
+        args: [fingerprint, label],
+      }),
+      invalidatesTags: () => ['DaemonKey'],
+      transformResponse: (response: any) => {
+        console.log('setLabel response', response);
```
Comment (paninaro) [inline packages/api-react/src/services/daemon.ts:66]: Remove?

File: packages/api-react/src/services/daemon.ts [hunk 2]
```diff
@@ -1,23 +1,100 @@
...
+      }
+    >({
+      query: ({ fingerprint }) => ({
+        command: 'deleteLabel',
+        service: Daemon,
+        args: [fingerprint, label],
+      }),
+      invalidatesTags: () => ['DaemonKey'],
+      transformResponse: (response: any) => {
+        console.log('deleteLabel response', response);
```
Comment (paninaro) [inline packages/api-react/src/services/daemon.ts:84]: Remove?

File: packages/api-react/src/services/daemon.ts [hunk 3]
```diff
@@ -1,23 +1,100 @@
...
+    deleteLabel: build.mutation<
+      boolean,
+      {
+        fingerprint: number;
+      }
+    >({
+      query: ({ fingerprint }) => ({
+        command: 'deleteLabel',
+        service: Daemon,
+        args: [fingerprint, label],
```
Comment (paninaro) [inline packages/api-react/src/services/daemon.ts:80]: Remove `label` argument?

File: packages/api-react/src/services/wallet.ts [hunk 4]
```diff
@@ -600,6 +601,9 @@ export const walletApi = apiWithTag.injectEndpoints({
       }),
       invalidatesTags: (_result, _error, { fingerprint }) => [
         { type: 'Keys', id: fingerprint },
+        { type: 'Keys', id: 'LIST' },
+        { type: 'DaemonKey', id: fingerprint },
+        { type: 'DaemonKey', id: 'LIST' },
```
Comment (paninaro) [inline packages/api-react/src/services/wallet.ts:606]: If these are necessary, should the same be applied to the `addKey` and `deleteAllKeys` mutations?

File: packages/core/package.json [hunk 5]
```diff
@@ -83,25 +88,30 @@
     "babel-plugin-styled-components": "2.0.2",
     "babel-plugin-transform-imports": "2.0.0",
     "cross-env": "7.0.3",
+    "cypress": "10.8.0",
+    "electron": "19.0.4",
```
Comment (paninaro) [inline packages/core/package.json:92]: I believe this is what Snyk was complaining about. If it's necessary, we can probably use 20.1.3 like we do in `packages/gui/package.json`

File: packages/core/package.json [hunk 6]
```diff
@@ -83,25 +88,30 @@
...
     "@chia/api-react": "1.0.0",
+    "@electron/remote": "2.0.8",
     "@lingui/core": "3.13.0",
     "@lingui/react": "3.13.0",
     "@mui/icons-material": "5.5.1",
     "@mui/lab": "5.0.0-alpha.75",
     "@mui/material": "5.5.2",
     "@mui/styled-engine-sc": "5.5.2",
     "@mui/styles": "5.5.1",
+    "electron": ">=19.0.0",
```
Comment (paninaro) [inline packages/core/package.json:114]: Maybe this too

File: packages/core/src/screens/SelectKey/SelectKeyRenameForm.tsx [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+        <TextField
+          name="label"
+          size="small"
+          disabled={!canSubmit}
+          data-testid="SelectKeyRenameForm-label"
+          onKeyDown={handleKeyDown}
+          InputProps={{
+            endAdornment: (
+              <InputAdornment position="end">
+                <Tooltip title={<Trans>Cancel Editation</Trans>}>
```
Comment (paninaro) [inline packages/core/src/screens/SelectKey/SelectKeyRenameForm.tsx:79]: Maybe just "Cancel" by itself?

File: packages/gui/playwright-report/index.html [hunk 8]
```diff
@@ -0,0 +1,62 @@
+
```
Comment (paninaro) [inline packages/gui/playwright-report/index.html:1]: This file was deleted in main. Do we need it? I was thinking we should just add it to .gitignore if it keeps coming back.

**Comment by paninaro** [comment]
Were the cypress inclusions intentional? If it's simple enough, I would probably put those in a separate branch/PR, but if not we can just keep it included in this PR.

File: packages/core/package.json (hunk 5 above)
Comment (seeden) [reply packages/core/package.json:92]: Rebased

File: packages/core/package.json (hunk 6 above)
Comment (seeden) [reply packages/core/package.json:114]: Updated to 20.1.3

File: packages/api-react/src/services/wallet.ts (hunk 4 above)
Comment (seeden) [reply packages/api-react/src/services/wallet.ts:606]: Nice catch, updated.

**Comment by seeden** [comment]
> Were the cypress inclusions intentional? If it's simple enough, I would probably put those in a separate branch/PR, but if not we can just keep it included in this PR.

I will create separate PR. Cypress was removed

**Review by paninaro — APPROVED** [review]
lgtm

**Pull request merged**